plumbing/transport/loader_test.go

```diff
@@ -0,0 +1,71 @@
+package transport
+
+import (
+	"os/exec"
+	"path/filepath"
+
+	"github.com/go-git/go-git/v5/storage/memory"
+
+	. "gopkg.in/check.v1"
```

Comment:
Due to the use of gocheck, I would either refrain from adding this test now, or migrate it to testify.

Reply:
I've retained plumbing/server/loader_test.go` until the work on the new transport tests begins